common/cards/alter-egos/single-use/anvil.ts

```diff
@@ -32,9 +32,14 @@ const Anvil: SingleUse = {
 		'Do 30hp damage to the Hermit directly opposite your active Hermit on the game board, and 10hp damage to each Hermit below it.',
 	hasAttack: true,
 	attackPreview: (game) => {
-		const targetAmount = getTargetHermits(game, game.currentPlayer).length - 1
-		if (targetAmount === 0) return '$A30$'
-		return `$A30$ + $A10$ x ${targetAmount}`
+		const targets = getTargetHermits(game, game.currentPlayer)
+		if (targets.length === 0) return '0'
```

Comment:
@alicetouhou Should this be '$A0$' or '0'

Reply:
$A0$ as it's a damage value. Thanks for asking!